### PR TITLE
Chore/daef 550 Refactor Cardano type declarations to type folder

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces refactored Cardano type declarations in type folder.

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@DominikGuzei this PR is pretty large and would be great if you could review it too ;)

**Comment by nikolaglumac** [comment]
@DominikGuzei @darko-mijic this is the issue I have detected while reviewing this PR: https://iohk.myjetbrains.com/youtrack/issue/DAEF-562
We should discuss it and solve it in a separate PR.

**Comment by DominikGuzei** [comment]
Thanks for adding the card @nikolaglumac, im reviewing this PR now

File: app/api/ada/getAdaAccounts.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
 // @flow
-import type { ApiAccounts } from 'daedalus-client-api';
+import type { AdaAccounts } from './types';
 import { request } from './lib/request';
 
-export const getAdaAccounts = (ca: string): Promise<ApiAccounts> => (
+export type GetAdaAccountsParams = {
+  ca: string,
+  walletId: string,
+  filePath: string,
```
Comment (DominikGuzei) [inline app/api/ada/getAdaAccounts.js]: This looks wrong to me … the `walletId` and `filePath` are not used anywhere?

File: app/api/etc/types.js [hunk 2]
```diff
@@ -1,14 +1,44 @@
+// @flow
+import BigNumber from 'bignumber.js';
+
+export type EtcTransactioPassphrase = string;
```
Comment (DominikGuzei) [inline app/api/etc/types.js]: There is a typo in `EtcTransactioPassphrase` 😉

File: features/send-money-to-receiver.feature [hunk 3]
```diff
@@ -24,6 +24,7 @@ Feature: Send Money to Receiver
       | balance  |
       | 0.000010 |
 
+  @watch
```
Comment (DominikGuzei) [inline features/send-money-to-receiver.feature]: Please remove

Comment (tomothespian) [reply features/send-money-to-receiver.feature]: Thanks @DominikGuzei . I will do fixes you mentioned 😉 . Tests failed only in one case which @nikolaglumac mention above.

File: app/api/etc/types.js (hunk 2 above)
Comment (tomothespian) [reply app/api/etc/types.js]: good eye 😉

**Comment by nikolaglumac** [comment]
@DominikGuzei Tomislav has implemented all requested changes. Merging this now...

**Pull request merged**